packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,30 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  &:not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: var(--gcds-link-underline-offset);
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-link-decoration-thickness);
+  }
+
+  &:hover {
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-link-hover-decoration-thickness);
+  }
+
+  &:focus {
+    background-color: var(--gcds-link-focus-background);
```

Comment:
I don't want to be annoying but would we be able to add a `border-radius` to the focus style to be consistent with the focus styles of other components.

Reply:
I think we have a lot of other components that don't have this, I will add it here, but maybe something we can add to the backlog to update? I just did a quick search for everything that has `&:focus`

Reply:
A lot of other components have the border radius applied before focus but a good example of this is in the [details component](https://github.com/cds-snc/gcds-components/blob/develop/packages/web/src/components/gcds-details/gcds-details.css#L27) (that's my goto component when I need to remember focus styles)

Reply:
Awesome, thanks Ethan!